addons/cssresources/src/css-resource-panel.tsx

```diff
@@ -5,7 +5,7 @@ import Eventtypes, { STORY_RENDERED } from '@storybook/core-events';
 import { EVENTS, PARAM_KEY } from './constants';
 import { CssResource } from './CssResource';
 
-interface CssResourcePanelProps {
+interface Props {
   active: boolean;
   api: {
     emit(event: any, data: any): void;
```

Comment:
is `event: any` or is it `event: () => any` ? 
or even `event: (...args) => any` in case it allows properties you don't know  yet?

Reply:
Hmm that should actually become `API` since there's a type for it now